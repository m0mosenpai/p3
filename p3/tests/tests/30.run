../solution/wsh tests/30.wsh 2>/dev/null
