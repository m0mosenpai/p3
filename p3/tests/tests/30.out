1
2
3
4
5
6
7
8
9
10
8
8
8
1) echo 10
2) echo 9
3) echo 8
4) echo 7
5) echo 6
6) echo 5
7) echo 4
8) echo 3
9) echo 2
10) echo 1
1) echo 10
2) echo 9
3) echo 8
4) echo 7
5) echo 6
6) echo 5
7) echo 4
8) echo 3
9) echo 2
10) echo 1
1) echo 10
2) echo 9
3) echo 8
4) echo 7
5) echo 6
1) echo 10
2) echo 9
3) echo 8
4) echo 7
5) echo 6
